docs/organizations/using-coding-standards.md

```diff
@@ -24,14 +25,23 @@ To create a coding standard for your organization:
 
 1.  Enter a unique name for the new coding standard.
 
-    Optionally, you can:
+    Optionally, you can set the new coding standard as your organization's [default coding standard](#set-default), so it applies automatically to new repositories.
 
-    -   Select a repository to use as a baseline for the new coding standard. This is useful if you already have a well-configured repository that you wish to use as a template.
-    -   Set the new coding standard as your organization's [default coding standard](#set-default), so it applies automatically to new repositories.
+    ![Creating a new coding standard](images/coding-standard-create.png)<!-- TODO ALA-1057 update image -->
 
-    ![Creating a new coding standard](images/coding-standard-create.png)
+1.  Choose whether to create the coding standard using a guided or advanced setup:
 
-1.  Click **Create coding standard**.
+    -   The **guided setup** lets you choose preset sensitivity levels for each issue type, turning on relevant patterns in bulk. It's ideal for quickly setting up a coding standard with minimal effort. You can always fine-tune the coding standard later using the advanced setup.
```

Comment:
Maybe we can add something like 'editing the standard at any time after creation'?
```suggestion
    -   The **guided setup** lets you choose preset sensitivity levels for each issue type, turning on relevant patterns in bulk. It's ideal for quickly setting up a coding standard with minimal effort. You can always fine-tune the coding standard later using the advanced setup, by editing it.
```